komorebi/src/border.rs

```diff
@@ -108,25 +108,23 @@ impl Border {
                 Self::create("komorebi-border-window")?;
             }
 
-            let mut should_expand_border = false;
+            let mut should_contract_border = true;
```

Comment:
Can you include some photographs and videos of the before and after of this change?